pygeoapi/templates/collections/coverage/index.html

```diff
@@ -0,0 +1,85 @@
+{% extends "_base.html" %}
+{% block title %}{{ super() }} {{ data['title'] }} {% endblock %}
+{% block crumbs %}{{ super() }}
+/ <a href="{{ data['collections_path'] }}">{% trans %}Collections{% endtrans %}</a>
+{% for link in data['links'] %}
+  {% if link.rel == 'collection' %} /
+    <a href="{{ data['dataset_path'] }}">{{ link['title'] | string | truncate( 25 ) }}</a>
+    {% set col_title = link['title'] %}
+  {% endif %}
+{% endfor %}
+/ <a href="{{ data['items_path']}}">{% trans %}Items{% endtrans %}</a>
+{% endblock %}
+{% block extrahead %}
+    <link rel="stylesheet" href="https://unpkg.com/leaflet/dist/leaflet.css"/>
+    <script src="https://unpkg.com/leaflet/dist/leaflet.js"></script>
+    <script src="https://unpkg.com/leaflet.imageoverlay.arrugator"></script>
+    <script src="https://unpkg.com/proj4@2.8.0/dist/proj4-src.js"></script>
+{% endblock %}
+
+{% block body %}
+  <section id="map"></section>
+  <section id="collection">
+      <h1>{{ data['title'] }}</h1>
+      <p>{{ data['description'] }}</p>
+  </section>
+<div class="col-md-8">
+	<div id="coverage-map"></div>
+</div>
+{% endblock %}
+
+{% block extrafoot %}
+
+    <script>
+    const map = L.map('coverage-map').setView([0,0], 0);
+    map.addLayer(new L.TileLayer(
+        '{{ config['server']['map']['url'] }}', {
+            maxZoom: 18,
+            attribution: '{{ config['server']['map']['attribution'] | safe }}'
+        }
+    ));
+
+    const bbox = {{ data.bbox | to_json | safe}};
+
+    /// TODO: How to get the proj.4 string for this CRS???!!
```

Comment:
The client-side functionality (which leverages `proj4js`) needs to know the proj4-style string for the CRS in order to warp the coverage image.

The current implementation works for some of my test data, in EPSG:4326 and EPSG:25833, but nothing else.

A possible approach would be something like 
```
    def _get_proj4_string(self, epsg_code):
       from osgeo import osr
       prj = osr.SpatialReference()
       prj.ImportFromEPSG(epsg_code)
 
       return prj.ExportToProj4().strip()
```